README.md

```diff
@@ -1,2 +1,55 @@
 # prj-jse-bookstore
-Desktop application to automate a book store
+Desktop application to automate a book store.
+
+## How it works?
+It's very simple to use, these are some instructions:
+
+**_Menu:_** 
+   - At the menu you can access the options for:
+
+**1. Products:**
+   - _Book:_ you can add, update and delete a book or list all the books.
+   - _DVD:_ you can add, update and delete a dvd or list all the dvds.
+   - _Hardware:_ select between a Cellphone or a Laptop, so you can add, update and delete a hardware or list all the hardwares in the category.
+   - _Magazine:_ you can add, update and delete a magazine or list all the magazines.
+   - _Return:_ always use to return to the previous menu.
+
+**2. Individuals:**
+   - _Customer:_ you can add, update and delete a customer or list all the customers.
+   - _Employee:_ you can add, update and delete an employee or list all the employees.
+   - _Return:_ always use to return to the previous menu.
+
+**3. Order:**
+   - Use to add an order at the book store.
+
+**4. Reports:**
+   - _Individuals:_ here you can generate some reports related to customers and employees.
+      - _Customers Filtered and Ordered by Name:_ insert a name or letter to search a customer, and generate a report filtered and ordered by customer name.
+      - _Employees Filtered by Name and Ordered by Highest Salary:_ insert a name or letter to search an employee, and generate a report filtered and ordered by the highest salary.
+      - _Employee Salary with Commission by Month:_ search an employee by its ID and the date that you want to generate your salary with commission report. 
+   - _Stock - Products to Replace:_ lists all products that don't have more stock.
+   - _Sales:_ here you can generate some reports related to sales.
+      - _Total Orders in a Month:_ insert the date that you want to know which orders were generated in this period and the amount.
+      - _Total Sales per Seller:_ insert the employee ID that you want to know how many sales he has made so far.
```

Comment:
"insert the employee ID to find out the amount of sales done so far by a given employee"